src/views/sponsorship.html

```diff
@@ -296,11 +308,11 @@ <h1 id="sponsorship-hack-cambridge-wordmark">Hack<br>Cambridge<br><span class="p
                 <div class="container a4">
                     <div class="one unit row">
                         <h5>
-                            Interested?<br><span class="secondary"><a href="mailto:sponsorship@hackcambridge.com">Contact our dedicated sponsorship team.</a></span>
+                            Interested?<br><span class="secondary">Contact our dedicated <a href="mailto:sponsorship@hackcambridge.com">sponsorship team &rarr;</a></span>
```

Comment:
I think we should keep the whole of the call to action as the link, since the link is for contacting the sponsorship team.  The arrow is a good idea if we stick with not showing the email address explicitly — maybe having an arrow and an underline on hover would be enough?

Another approach could be making the email address explicit, for example `Contact our dedicated sponsorship team at <a href="...">sponsorship@hackcambridge.com</a>`.  This might make it more clear that clicking the link creates an email to the sponsorship team.

Reply:
You might be interested in https://hc2018admin.slack.com/archives/C4L4WBB5M/p1504301119000073

Reply:
Ah, I missed that, thanks.  Personally, having half the text as the link still looks off to me, but it's your call.

Reply:
I think it's the best solution for a rather complicated problem